### DISCUSSION TITLE
List 5 movie titles contains a prime number >5 🍿🎞️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1020] [Galaxy Trivia] [answered]
**Bonus:** Extra points for Oscar winners 🎥🏆

**Comment by peterbaccaro** [comment]
1. **TWO SOLDIERS** (2003) - 1 OSCAR
2. **THE LORD OF THE RINGS: THE TWO TOWERS** (2002) - 2 OSCARS
3. **TWO DISTANT STRANGERS** (2020) - 1 OSCAR
4. **THREE BILLBOARDS OUTSIDE EBBING, MISSOURI** (2017) - 2 OSCARS
5. **SEVEN BRIDES FOR SEVEN BROTHERS** (1954) - 1 OSCAR

**Comment by thuongtruong1009** [comment]
Here are five movie titles that contain a prime number greater than 5:

1. "12 Years a Slave" - This movie won the Academy Award for Best Picture in 2014.
2. "The Hurt Locker" - This movie won the Academy Award for Best Picture in 2009.
3. "Argo" - This movie won the Academy Award for Best Picture in 2012.
4. "Million Dollar Baby" - This movie won the Academy Award for Best Picture in 2005.
5. "Spotlight" - This movie won the Academy Award for Best Picture in 2015.

These movie titles contain the prime numbers 12, 7, 5, and 2 respectively.

**Comment by tomerf** [comment, marked as answer]
1. Se7en (1995)
2. Ocean's Eleven (2001)
3. Ocean's Thirteen (2007)
4. 127 Hours (2010)
5. The Edge of Seventeen (2016)

**Comment by virajchandra51** [comment]
Se7en (1995)
Ocean's Eleven (2001)
Ocean's Thirteen (2007)
127 Hours (2010)
The Edge of Seventeen (2016)
